platform/android/java/src/org/godotengine/godot/Godot.java

```diff
@@ -407,16 +408,28 @@ private void initializeGodot() {
 		GodotLib.io = io;
 		mSensorManager = (SensorManager)getSystemService(Context.SENSOR_SERVICE);
 		mAccelerometer = mSensorManager.getDefaultSensor(Sensor.TYPE_ACCELEROMETER);
-		mSensorManager.registerListener(this, mAccelerometer, SensorManager.SENSOR_DELAY_GAME);
 		mGravity = mSensorManager.getDefaultSensor(Sensor.TYPE_GRAVITY);
-		mSensorManager.registerListener(this, mGravity, SensorManager.SENSOR_DELAY_GAME);
 		mMagnetometer = mSensorManager.getDefaultSensor(Sensor.TYPE_MAGNETIC_FIELD);
-		mSensorManager.registerListener(this, mMagnetometer, SensorManager.SENSOR_DELAY_GAME);
 		mGyroscope = mSensorManager.getDefaultSensor(Sensor.TYPE_GYROSCOPE);
-		mSensorManager.registerListener(this, mGyroscope, SensorManager.SENSOR_DELAY_GAME);
 
 		GodotLib.initialize(this, io.needsReloadHooks(), getAssets(), use_apk_expansion);
 
+		String sensor_delay_string = GodotLib.getGlobal("input_devices/sensors/sensor_delay");
```

Comment:
This call seems to be crashing android for me, I don't know if getGlobal is unavailable at this point of time, or whether our project file hasn't been loaded yet, or if there is some other reason... Anyone have any ideas?

Reply:
no idea but I got several reports related to it.

Reply:
I'm pretty much in the dark how this whole thing ties together, it all feels pretty messy to me. So yeah I still don't know if I'm doing this too early, if there is a better place to call this from, or if there is something wrong with getting the setting.

it's a shame because this is the final bit to get simple VR going well on Android :)

Reply:
The call is crashing because [GodotLib.getGlobal](https://github.com/godotengine/godot/blob/25c1363a1167cc5833111e78e33b2561f1ceb76c/platform/android/java/src/org/godotengine/godot/GodotLib.java#L69) [relies](https://github.com/godotengine/godot/blob/25c1363a1167cc5833111e78e33b2561f1ceb76c/platform/android/java_glue.cpp#L1488) on [ProjectSettings](https://github.com/godotengine/godot/blob/25c1363a1167cc5833111e78e33b2561f1ceb76c/core/project_settings.h#L42) which is not [initialized](https://github.com/godotengine/godot/blob/25c1363a1167cc5833111e78e33b2561f1ceb76c/main/main.cpp#L346) until [GodotLib.setup](https://github.com/godotengine/godot/blob/25c1363a1167cc5833111e78e33b2561f1ceb76c/platform/android/java/src/org/godotengine/godot/GodotLib.java#L49) has been [invoked here](https://github.com/godotengine/godot/blob/25c1363a1167cc5833111e78e33b2561f1ceb76c/platform/android/java/src/org/godotengine/godot/Godot.java#L311).

I'm adding another comment about how to possibly resolve the issue.